static/js/fieldUpdater.js

```diff
@@ -1,3 +1,5 @@
+import { ajaxDelete, ajaxUpdate } from "./ajaxUpdate.js";
```

Comment:
I was playing around with async imports, looking to understand them. Can you explain the reason you changed this?

Reply:
Collects the imports into one spot at the top rather than throughout the code. Named imports are already available as a variable to call, so there's no need to set them with `let/const`.

Reply:
So I don't mind that, it's just an import declaration in a different syntax and place.
The differences I think may be important may be
1. static imports are better for performance of JS, as can be statically analysed
2. dynamic imports don't request the asset until needed, function is called, request is made for code file

I'm not clear on exaclty what happens requestwise with the import as is. I assume it automatically makes a request for ajaxUpdate on pageload, but have not checked.